scripts/simex

```diff
@@ -300,6 +305,68 @@ def do_experiments_list(args, as_default_subcmd=False):
 			return colors['red']
 		return ''
 
+	def show_compact_list():
+
+		def print_experiment_statistics():
+
+			def _get_table_entries(status_list, include_status_string=False):
+				entry_list = []
+				for s in status_list:
+					if status_dict[s] > 0:
+						prefix = ''
+						if include_status_string:
+							prefix = str(s) + ': '
+
+						entry_list.append(prefix + str(status_dict[s]) + '/' + str(num_runs))
+
+				if len(entry_list) == 0:
+					return entry_list.append('')
+
+				return entry_list
```

Comment:
This is a bit ugly, in particular since we return an array and often only need one argument. Why don't you let this function act only on a single status and call it multiple times below (e.g. inside a list comprehension) when you need the result on multiple statuses?

Reply:
As discussed we will leave the code as it is for now, since it easily handles zero-entries.
I only removed the `if len(entry_list) == 0` case as it is handled by the `fillvalue` in `zip_longest`.